packages/components/src/components/select/select-option/select-option-styles.ts

```diff
@@ -48,10 +55,11 @@ export const getOptionStyles = (theme: Theme): Styles => {
 
   return {
     option: {
+      fontWeight: fontWeightRegular,
       display: 'flex',
-      // justifyContent: 'space-between', // TODO: Commenented out
       gap: '12px',
       padding: `${spacingStaticSmall} 12px`,
+      paddingLeft: `var(${cssVariableSelectPaddingLeft}, 12px)`,
```

Comment:
Does padding **left** work in rtl context?

Reply:
No. Has to be fixed 👍🏼 
<img width="951" alt="image" src="https://github.com/user-attachments/assets/1e6aa69c-ccbc-498b-987d-4209b9847f59">